test/indexTest.js

```diff
@@ -0,0 +1,13 @@
+const sinon = require( 'sinon' );
+const helpers = require( './helpers' );
+const chai = require( 'chai' );
+const spies = require( 'chai-spies' );
+
+chai.use( spies );
+
+describe( "main.js", () => {
+  it("passes a 'smoke test'", () => {
```

Comment:
is there another way to describe this? Not sure students will know what "smoke test" means if they see this. Otherwise this looks great to me!

Reply:
Yeah, I was struggling to come up with a description that recognizes that the tests won't cover everything. Maybe better as a mention in the Readme instead of in the test description, and then the test can just say something like "contains a hidden modal"

Reply:
Updated with a note in the Readme.

Reply:
Yeah I think that's good - thanks Rob!